zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -184,12 +185,18 @@ public static synchronized ZeppelinConfiguration create() {
     return conf;
   }
 
+  public Map<String, String> getProperties() {
+    return this.properties;
+  }
+
   public static void reset() {
     conf = null;
   }
 
   public void setProperty(String name, String value) {
-    this.properties.put(name, value);
+    if (name != null && value != null) {
```

Comment:
Maybe `StringUtils.isNoneBlank()`  is better in this place, because with this code you can add keys and/or values like `" "`.

Reply:
Fixed